calculator/calc.cpp

```diff
@@ -0,0 +1,79 @@
+#include<iostream>
+using namespace std;
+int main()
+{
+    double a,b,c;
+    int n;
+
+    cout<<"\n$$$ ENTER NUMERICAL VALUES ONLY TO PERFORM OPERATION AS IT IS CALCULATOR: $$$"<<endl;
+    cout<<"----------------------------------------------------------------------------------------\n";
+
+
+    cout<<"Enter First Number"<<endl;
```

Comment:
Ask for user's choice first and then ask for input numbers

Reply:
Resolved